subsystem-test/tests/src/test/java/org/jboss/as/subsystem/test/transformers/subsystem/simple/VersionedExtensionCommon.java

```diff
@@ -128,6 +128,10 @@ protected void performRuntime(OperationContext context, ModelNode operation, Mod
 
     private static class BasicAttributeWriteHandler extends AbstractWriteAttributeHandler<Void> {
 
+        protected BasicAttributeWriteHandler() {
+            super(List.of());
+        }
+
```

Comment:
@pferraro This class seems wrong; it seems to be adding a new call to a deprecated c'tor instead of removing the existing one.

Reply:
Hmm - not sure what I was thinking here.  Will fix.